src/Stryker.Core/Stryker.Core/TestRunners/VsTest/VsTestContextInformation.cs

```diff
@@ -278,24 +240,22 @@ private string GenerateRunSettingsForDiscovery()
 
             return $@"<RunSettings>
  <RunConfiguration>
-  <CollectSourceInformation>true</CollectSourceInformation>
-  <MaxCpuCount>{Options.Concurrency}</MaxCpuCount>
-  <DesignMode>false</DesignMode>
+  <MaxCpuCount>{Math.Max(1, Options.Concurrency)}</MaxCpuCount>
```

Comment:
Curious why you added this, because Concurrency is not allowed to be less than 1: https://github.com/stryker-mutator/stryker-net/blob/master/src/Stryker.Core/Stryker.Core/Options/Inputs/ConcurrencyInput.cs

Reply:
Yes, but I ran into an (intermediate) bug where `Concurrency` value was set to 0. Which results in the infamous VsTest bug where it does not start at all but we are not aware, so Stryker can only fail badly. A bit like the previous case with negative timeout values.
So I though that a bit of defensive programming was ok here.